### PR TITLE
Better OS Detection and Notification

### PR DESCRIPTION
**Pull request by jfryman**
This PR adds some basic information on supported operating systems, and points the user to a place where they can get more information if need be.

Stubs for the KB articles, will be filled out shortly.

### CODE REVIEW

File: script/check-os-compat [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+compatable_os() {
+    echo "Passed OS Detection..."
+    exit 0
+}
+
+incompatable_os() {
+    echo ""
+    echo "OH NO! Y U NO WORK?!"
+    echo ""
+    echo "Well, bummer. It seems that you have found yourself at the end of a"
```
Comment (arm4b) [inline script/check-os-compat:19]: lol, you rock with such things :smile:

File: script/check-os-compat [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    echo "https://stackstorm.reamaze.com/articles/adding-os-support-to-all-in-one-installer"
+    echo "and see if you can lend us a hand."
+    echo ""
+    echo "Don't forget to let us know if we can help out in any other way. Come find"
+    echo "us using your favorite communication mechanism, and we'll be there with a"
+    echo "hot cup of cocoa and some tips on how to get some automation happening!"
+    echo ""
+    echo "Email: support@stackstorm.com"
+    echo "IRC: irc.freenode.net/#stackstorm"
+    echo "Slack: http://stackstorm.com/community-signup"
```
Comment (lakshmi-kannan) [inline script/check-os-compat]: https

**Comment by lakshmi-kannan** [comment]
Nice!

File: script/check-os-compat [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+incompatable_os() {
+    echo ""
+    echo "OH NO! Y U NO WORK?!"
+    echo ""
+    echo "Well, bummer. It seems that you have found yourself at the end of a"
+    echo "long cavern. Ahead of you is darkness. You can hear the wind blowing"
+    echo "across your left ear"
+    echo ""
+    echo "You are running on an unsupported OS. Good news there is a high"
```
Comment (manasdk) [inline script/check-os-compat]: high chance perhaps?

**Comment by manasdk** [comment]
+1

**Comment by Kami** [comment]
Nice :+1:

**Pull request merged**